src/util/equalArrays.ts

```diff
@@ -1,7 +1,5 @@
-//@ts-nocheck
-
 /** Equality for lists of lists. */
-export const equalArrays = (a, b) =>
+export const equalArrays = (a: unknown[], b: unknown[]): boolean =>
```

Comment:
Does `a: []` work, or is `unknown` required?

Reply:
Afaik, that doesn't work. We need to specify the type of items.

Reply:
Did you try it?

Reply:
Yes. Please check [this](https://www.typescriptlang.org/play/#code/MYewdgzgLgBA5gUygGQWOUAWMC8MAUAhgFwwDaAugJS4B8MhAdADZoaYDcAUF6JCKxYg4+RCjZZ8ZAIwAaAEzUqQA)

Reply:
Got it, thanks!